kinetic/distribution.yaml

```diff
@@ -9514,6 +9514,16 @@ repositories:
       url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
       version: master
     status: maintained
+  orb_slam2_with_maps-odom:
```

Comment:
Using `-` in a repository name is discouraged, would you consider renaming it?

Reply:
OK,I will change it.